results.tex

```diff
@@ -28,16 +28,21 @@ \subsection{Underlying technology}
 Our goals include (1) to list regular Machine Learning algorithms that are commonly used in a distributed setting; (2) to find algorithms to determine the best parameters for the aforementioned algorithms; (3) to find a balance between computation time, communication overhead, and accuracy; and (4) to minimize the number of bits sent over the network, so that the system is not bottlenecked by network bandwidth.\\
 Designing a generic system that enables distribution of  regular Machine Learning efficiently is challenging, because every algorithm has its own communication patterns \cite{Jia14}\cite{Newman09}\cite{Rich13}\cite{Smola10}\cite{Takac13}\cite{Tsi12}.
 
-\subsubsection{Machine Learning algorithms}
-Machine Learning algorithms learn to make predictions or decisions based on data. Every Machine Learning algorithm has its own pros and cons. Machine Learning algorithms can be divided into categories based on some of their characteristics:
+\subsubsection{Machine Learning algorithms / parameter optimization}
+Machine Learning algorithms learn to make predictions or decisions based on data. Every Machine Learning algorithm has its own pros and cons. These can be divided into categories based on some of their characteristics:
 \begin{itemize}
 	\item \textbf{Feedback}, the type of feedback that is given to the algorithm while learning
 	\item \textbf{Goal}, the type of output that is generated
 	\item \textbf{Type}, the architecture used to generate output
 	\item \textbf{Method}, the nature of model evolution that occurs when given feedback
+	\item \textbf{Ensemble Methods}, the combination of algorithms to improve the accuracy
 \end{itemize}
+We'll first look at the ML algorithms themselves and then at other algorithms that optimize the parameters of these ML algorithms.
 
-\paragraph{Feedback}
+\paragraph{Machine Learning categories}
+\subparagraph{ML category: Feedback}
+.\\
+To train an algorithm, it has to get feedback so that the algorithm can improve. There are several different types of feedback:
```

Comment:
"get" -> "receive"

Reply:
Also, two "algorithm"s very close to each other.

Reply:
It seems that "get feedback" is more common, see http://context.reverso.net/translation/dutch-english/feedback+krijgen